packages/core/src/components/tooltip/tooltip.scss

```diff
@@ -19,7 +19,6 @@
   font-weight: 400;
   line-height: typography.$line-height-base;
   line-break: auto;
-  text-align: left; // Fallback for where `start` is not supported
```

Comment:
Waarom kan/is deze weg?

Reply:
Omdat die fallback wat achterhaald is: https://caniuse.com/?search=text-align%3A%20start

Reply:
Ok! Extraatje dus. Dank